templates/typescript_gapic/package.json.njk

```diff
@@ -0,0 +1,40 @@
+{
+  "repository": "googleapis/nodejs-{{ api.naming.productName.toKebabCase() }}",
+  "name": "{{ api.naming.productName.toKebabCase() }}",
+  "version": "0.1.0",
+  "author": "Google LLC",
+  "description": "GAPIC {{ api.naming.productName }} client for Node.js",
+  "main": "build/src/index.js",
+  "files": [
+    "protos",
+    "build/src",
+    "!build/src/**/*.map",
+    "AUTHORS",
+    "COPYING"
+  ],
+  "dependencies": {
+    "google-gax": "googleapis/gax-nodejs"
+  },
+  "devDependencies": {
+    "@types/mocha": "^5.2.5",
+    "@types/through2": "^2.0.34",
```

Comment:
We should never need `through2`

Reply:
That will be updated - I'm going to backport all the gapic-generator fixes here.